### DISCUSSION TITLE
p/8142d81f162f4603b4cbf886c3035de6/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#16] [Announcements]
# p/8142d81f162f4603b4cbf886c3035de6/

生活这周开始尝试了有计划的生活，使用日程规划明天的非工作时间。从去年开始，已经开始感觉到没有计划随心做事已经很难提升自己了，直到这周才开始真正实践。计划在工作日执行得还不错，但周末几乎完全失败了。一方面，没有料到一些插入的时间会那么花时间，例如公司的体检从今天 9 点半做到了 12 点，比吾辈想象中要花的时间多不少。另一方面，对于周末，仍未有某种规律的感觉，对之的时间感觉与工作日并不相同，即便花费

https://blog.rxliuli.com/p/8142d81f162f4603b4cbf886c3035de6/

**Comment by moeyua** [comment]
多邻国好诶，要加好友吗

**Reply by moeyua** [reply]
https://www.duolingo.com/profile/MoeyuaEvod?via=share_profile

**Reply by rxliuli** [reply]
> 多邻国好诶，要加好友吗

被判定为国内用户，表示无法使用好友功能，也许你能添加吾辈？id: rxliuli

**Reply by moeyua** [reply]
国区现在似乎没有好友功能，不过如果没有绑定国内手机号的话还是可以转移到外区的

**Reply by rxliuli** [reply]
算了，这也怪不了 app，只能说洼地就是本身如此，社交本身吾辈更多活跃于 twitter，感兴趣可以看看，twitter id: rxliuli

**Reply by moeyua** [reply]
来了！
